test-runner-frontend/src/Components/TestRunsTable.js

```diff
@@ -0,0 +1,241 @@
+}
+
+export default function TestRunsTable({childToParentCheckBox, checkBoxState, testResults, rowsF, sortBenchmarks, sortDate}) {
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(10);
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(+event.target.value);
+    setPage(0);
+  };
+
+  const handleChange = (value) => { 
+   // setCheckBoxSelection(checkBoxSelection.filter(test => test === value));
+   childToParentCheckBox(value); 
+  }; 
+  
+  
+
+  /*
+  const handleChange = (value) => { 
+    console.log(true);
+   };
+   */
+   
+
+  return (
+    
+    <Paper sx={{ width: "100%", marginTop: checkBoxState == false ? "2.5%" : "1.2%" }} elevation={12}>
+      <TableContainer sx={{ maxHeight: checkBoxState == false ? "54vh" : "48vh" }} >
+        <Table stickyHeader>
+          <TableHead>
+
+          
+
+            <TableRow>
+              {columns.map((column) => (
+                <TableCell
+                  key={column.id}
+                  align={column.align}
+                  style={{ top: 3, minWidth: column.minWidth }}
+                  //align= "left"
+                >
+                {
+                  column.label === "Date" ?
+                    <Stack direction="row" spacing={4}>
+                        <div><b>{column.label}</b></div>
+                        <IconButton color="primary" sx={{top: "13%", position: "absolute"}} onClick={() => {sortDate()}}>
+                            <ArrowUpwardIcon />
+                        </IconButton>
+                    </Stack>
+                  /*  :
+                    column.label === "Benchmarks" ?
+                    <Stack direction="row" spacing={11}>
+                    <div><b>{column.label}</b></div>
+                    <IconButton color="primary" sx={{top: "13%", position: "absolute"}} onClick={() => {sortBenchmarks()}}>
+                        <ArrowUpwardIcon />
+                    </IconButton>
+                    </Stack>
+                  */
+                    :
+                  <b>{column.label}</b>
+                }
+                </TableCell>
+              ))}
+            </TableRow>
+          </TableHead>
+          <TableBody>
+          
+
+            {rowsF
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              .map((row) => {
+                return (
+                  <TableRow hover role="checkbox" tabIndex={-1} key={row.id} sx={{maxHeight: "70px", height: "70px"}}>
+                     
+                    
+                    {columns.map((column) => {
+                      const value = row[column.id];
+                      return (
+                        
+                        <TableCell key={column.id} align={column.align}>
+                            
+                        {
```

Comment:
ternary statements can sometimes be a little hard to read. especially when they're big like this. can we please look at splitting these into small components? doing something like this should make it a little easier to read:
```jsx
{columns.map((column) => {
  const value = row[column.id];

	let cell = null;
	if(column.id === "improvedVsRegressed"){
		cell = <ComparisonCell />
	} else if(column.id !== "id"){
		cell = value
	// etc
	}

  return (
	  <TableCell key={column.id} align={column.align}>
	    {cell}
	  </TableCell>
  )
})}
```

Reply:
I didn't know we could use if else conditions, that's super cool! I lalready made the change, thank you!